### PR TITLE
Make killEmulator method work for current emu

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: lib/tools/system-calls.js [hunk 1]
```diff
@@ -546,13 +546,20 @@ systemCallMethods.killAllEmulators = async function () {
...
+      await this.adbExec(['emu', 'kill']);
+      log.info(`successfully killed emulator '${avdName}'`);
+    } else {
+      log.info(`no avd with name '${avdName}' running. skipping kill step.`);
+    }
   } else {
-    log.info(`no avd with name '${avdName}' running. skipping kill step.`);
+    // killing the current avd
+    log.debug(`killing avd with id '${this.curDeviceId}'`);
+    await this.adbExec(['emu', 'kill']);
```
Comment (mykola-mokhnach) [inline lib/tools/system-calls.js]: should we check if the emu is running first?

File: lib/tools/system-calls.js [hunk 2]
```diff
@@ -546,13 +546,20 @@ systemCallMethods.killAllEmulators = async function () {
...
-    await this.adbExec(['emu', 'kill']);
-    log.info(`successfully killed emulator '${avdName}'`);
+  if (util.hasValue(avdName)) {
+    log.debug(`killing avd '${avdName}'`);
+    let device = await this.getRunningAVD(avdName);
+    if (device) {
+      await this.adbExec(['emu', 'kill']);
+      log.info(`successfully killed emulator '${avdName}'`);
+    } else {
+      log.info(`no avd with name '${avdName}' running. skipping kill step.`);
```
Comment (mykola-mokhnach) [inline lib/tools/system-calls.js]: We could probably return false or true based on the fact that emulator has been killed or not

File: lib/tools/system-calls.js (hunk 1 above)
Comment (imurchie) [reply lib/tools/system-calls.js]: Sure. It is checked in the other section, too.

File: lib/tools/system-calls.js [hunk 3]
```diff
@@ -546,14 +546,24 @@ systemCallMethods.killAllEmulators = async function () {
  * @param {string} avdName - The name of the emulator to be killed.
```
Comment (mykola-mokhnach) [inline lib/tools/system-calls.js]: return description is missing. Also it worth to mention about the argument is optional

Comment (imurchie) [reply lib/tools/system-calls.js]: This is why I hate docstrings.

**Pull request merged**